tools/deb/istio-start.sh

```diff
@@ -42,8 +42,12 @@ ISTIO_SYSTEM_NAMESPACE=${ISTIO_SYSTEM_NAMESPACE:-istio-system}
 # The default matches the default istio.yaml - use sidecar.env to override this if you
 # enable auth. This requires node-agent to be running.
 ISTIO_PILOT_PORT=${ISTIO_PILOT_PORT:-8080}
-ISTIO_CP_AUTH=${ISTIO_CP_AUTH:-NONE}
 
+# If set, add the flag
+CONTROL_PLANE_AUTH_POLICY=""
```

Comment:
any concerns about changing the name of the variable? Is it ever passed into the script?

Reply:
It was not working - and not mentioned in sidecar.env. The script was failing.